### PR TITLE
Coinmate

### PR DESCRIPTION
**Pull request by stachon**
Hello, I have implemented support for Coinmate exchange https://coinmate.io

Best regards, 
Martin Stachon

### CODE REVIEW

**Comment by timmolter** [comment]
@stachon Great! There are merge conflicts. Can you make sure to update your fork's branch and push the changes? This PR will then automatically update.

File: xchange-coinmate/pom.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            <exclusions>
+                <!-- because the pusher jar has its own fork of Java-Websocket -->
+                <exclusion>
+                    <groupId>org.java-websocket</groupId>
+                    <artifactId>Java-WebSocket</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
```
Comment (timmolter) [inline xchange-coinmate/pom.xml]: Is pusher really a dependency for this module?

Comment (stachon) [inline xchange-coinmate/pom.xml]: Thanks, I have removed the dependency

**Comment by stachon** [comment]
Thanks, I have updated by branch.

**Comment by timmolter** [comment]
I don't understand why Travis CI is running the integration tests for your PR and it doesn't for other people's PRs. Any ideas? If you run `mvn clean test` on your fork, does it pass all unit tests?

**Comment by stachon** [comment]
This is because I forgot to change skipTests back to true in pom.xml. (To let NetBeans run tests from the IDE). Sorry about that!

**Pull request merged**